application-mocca-calendar-ui/src/main/resources/MoccaCalendar/Code/Macro.xml

```diff
@@ -1938,10 +1938,7 @@ Mocca Calendar Macro
       <contentType>No content</contentType>
     </property>
     <property>
-      <defaultCategories/>
-    </property>
-    <property>
-      <defaultCategory>Content</defaultCategory>
+      <defaultCategories>Content</defaultCategories>
```

Comment:
This might break the installation.

<defaultCatefories> was migrated to a collection element. Which means that it will have children elements.

Doing an export from 14.10 gives me this xml.
```
    <property>
      <defaultCategories>
        <value>Content</value>
      </defaultCategories>
    </property>
 ```